src/Place/ReadModel/History/Log.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace CultuurNet\UDB3\Place\ReadModel\History;
+
+use DateTime;
+use JsonSerializable;
+
+class Log implements JsonSerializable
```

Comment:
This class is not actually used by the projector. Personally I'd prefer not to duplicate it and move the one from event to a shared namespace instead, since it's important that both the event and place history have the same JSON structure, and we'd be duplicating that.